README.md

```diff
@@ -119,7 +122,7 @@ The following features of dbt are not implemented on Athena:
 
 * Quoting is not currently supported
   * If you need to quote your sources, escape the quote characters in your source definitions:
-  
+
   ```yaml
   version: 2
 
```

Comment:
Could you add a small line to specifcy how to install pre-commit locally? helpfull for contributors.
e.g.
```
* Install pre-comming via `pip install pre-commit`
* Run `pre-commit install`
```

Reply:
I added a `Contributing` section.